metricbeat/module/system/_meta/config.full.yml

```diff
@@ -4,10 +4,10 @@
     - cpu
 
     # System Load stats
-    - load
+    #- load
```

Comment:
Yeah, I'll revert this. For some reason I thought we removed all of the OS specific `before-build` changes.